elements/rh-navigation-secondary/demo/deprecated.html

```diff
@@ -0,0 +1,197 @@
+<link rel="stylesheet" href="demo.css">
```

Comment:
nice thinking adding this demo for QA

Reply:
I've approved the changes so far and leave this comment thread open for you to resolve-and-merge at your leisure